zkstack_cli/crates/zkstack/src/commands/chain/init/configs.rs

```diff
@@ -26,7 +26,7 @@ use crate::{
 };
 
 pub async fn run(args: InitConfigsArgs, shell: &Shell) -> anyhow::Result<()> {
-    let ecosystem_config = EcosystemConfig::from_file(shell)?;
+    let ecosystem_config = ZkStackConfig::ecosystem(shell)?;
```

Comment:
we shouldn't use it here. 
We need it only for port reallocation. 
It'd be better to pass zkstack config here and if  ecosystem is available do reallocation and if not, use default one

Reply:
We also need ecosystem for contracts config, this is fixed in a next PR:
https://github.com/matter-labs/zksync-era/pull/3231